deploy/manifests/dev-v2/us-east-2/cluster/storetheindex/flux-cd.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: GitRepository
+metadata:
+  name: storetheindex
+spec:
+  interval: 5m
+  url: https://github.com/filecoin-project/storetheindex.git
+  ref:
+    branch: main
+  secretRef:
+    name: github-auth
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: storetheindex-tenant
+spec:
+  # serviceAccountName: flux
```

Comment:
Can I ask you to expand on this please?

Reply:
I need to develop the tenant level restrictions. This flag is not required currently because I need complete the restrictive flow and it's currently disabled.
https://github.com/filecoin-project/gitops-root/issues/191

You can see that the flow is currently restricted here.
https://github.com/filecoin-project/gitops-root/blob/a71686bd1dc8837f0beac8b7af21e78aefa8e821/.weave-gitops/clusters/dev-v2/system/flux-system/kustomization.yaml#L13

I should complete this today/tomorrow.